src/views/core/members/UpdateProfile.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+  <div class="tile is-ancestor ">
+    <div class="tile is-child">
+
+      <template>
+        <div class="notification is-danger">
+          <div class="content">
+            <strong>Your profile has the following issues and you have to fix them before using the system:</strong>
+            <ul>
+              <li v-for="(error, key) in validationErrors" :key="key"><strong>{{ key }}</strong>: {{ error}}</li>
+            </ul>
+          </div>
+        </div>
+      </template>
+
+      <!-- <form @submit.prevent="saveUser()" v-if="validationErrors.email">
```

Comment:
Why are these comments here?

Reply:
we'll need this again when we'll add extra validation here that requires re-saving the user (email re-saving is different endpoint), but not for now, so I've commented it out for now.